apps/experiments/export.py

```diff
@@ -10,9 +10,23 @@ def _parse_tags(tags: list[Tag]) -> str:
     return ", ".join([t.name for t in tags])
 
 
+def _parse_comments(user_comments: list[UserComment]) -> str:
```

Comment:
nit: also rename the function
```suggestion
def _format_comments(user_comments: list[UserComment]) -> str:
```